### ISSUE TITLE
Adding Ctrl + P feature

### ISSUE THREAD

**Issue by shirshak55** [httptoolkit/httptoolkit#185]
We can discuss which shortcut should we use but the main idea is to add features like CTRL + P which gives an interface as we can see in the sublime text, visual studio code.

I would specifically like to add following features:
1. Ability to move to UI parts like Settings/Home Page or Dashboard etc.
2. Ability to move to certain sections like Proxy Config etc.
3. Ability to add selected request to the Mock section.

**Comment by pimterry** [comment]
Yes, this would be very cool! I think https://www.npmjs.com/package/react-command-palette is the most popular implementation for this, that would work.

It would need a little styling, to match the colours & general styles of HTTP Toolkit. I think starting with "jump to X" commands would be good and should be easy to do. "Launch [interceptor]" commands to quickly open another browser, terminal or similar from anywhere could be useful too.

For things like "Add selected request as mock", you'd need to be able to have commands that are only available when a request is currently selected, and hidden the rest of the time. That's presumably not impossible, but I'm not sure exactly how it would work, needs some investigation.
